scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -42,6 +42,14 @@ public class CompileTargets
         private static string DotnetHostFxrBaseName => $"{Constants.DynamicLibPrefix}hostfxr{Constants.DynamicLibSuffix}";
         private static string HostPolicyBaseName => $"{Constants.DynamicLibPrefix}hostpolicy{Constants.DynamicLibSuffix}";
 
+        public static readonly string[] PkgProjects = new[]
```

Comment:
I agree it is confusing, but this is already an issue with the repo, since that is the name used for these packages elsewhere:

https://github.com/dotnet/cli/blob/rel/1.0.0/scripts/dotnet-cli-build/CompileTargets.cs#L149
https://github.com/dotnet/cli/tree/rel/1.0.0/pkg

I tried thinking of better names, but failed. Also, it would be confusing to name this variable something that didn't align with "PackagePkgProjects"